src/lib/openjp2/jp2.c

```diff
@@ -1995,6 +1995,10 @@ OPJ_BOOL opj_jp2_setup_encoder(opj_jp2_t *jp2,
             jp2->enumcs = 17;    /* greyscale */
         } else if (image->color_space == 3) {
             jp2->enumcs = 18;    /* YUV */
+        } else if (image->color_space == 4) {
```

Comment:
would you mind using the OPJ_CLRSPC_xxxx enumeration values (for those new entries as well as for the 3 above existing ones?)